artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/impl/journal/AbstractJournalStorageManager.java

```diff
@@ -193,7 +193,7 @@ public static JournalContent getType(byte type) {
 
    protected final Map<SimpleString, PersistedAddressSetting> mapPersistedAddressSettings = new ConcurrentHashMap<>();
 
-   protected final Set<Long> largeMessagesToDelete = new HashSet<>();
+   protected final Map<Long, LargeServerMessage> largeMessagesToDelete = new ConcurrentHashMap<>();
```

Comment:
It is possible to use a primitive version of the map ie using primitive longs instead of boxed types

Reply:
@franz1981 modified, please review, Thanks.